cypress/e2e/laManager/manageServices/viewService-laMan.spec.js

```diff
@@ -0,0 +1,99 @@
+describe('LA Man - manage services - view services list', () => {
+    beforeEach(() => {
+        // Given I am logged in as DFE admin
+        cy.visit('/')
+        cy.integrationLogin('laman')
+
+        // And I am on the view services page
+        cy.navigateToViewServicesPage()
+
+    })
+
+    it('displays a list of services', () => {
+        const expectedPageHeading = "Tower Hamlets Council services";
+        const expectedResultsHeading = "Services";
+
+
+        // Then the page heading is displayed correctly
+        cy.checkPageHeading("h1", expectedPageHeading);
+
+        // And the results heading is displayed correctly - the xpath could not be helped there are no ids/data test ids
+        cy.checkPageHeading('#results > div > div > div > table > thead > tr > th:nth-child(1) > a', expectedResultsHeading);
+
+        // And the results are displayed
+        cy.get('[id="results"]').should('exist');
+    })
+
+    it('the save button should not be displayed', () => {
+
+
+        // Then the save button is not displayed
+        cy.get('#main-content > div > div > form > button').should('not.exist');
+    })
+
+    it('can apply filters on results', () => {
+
+        // When I apply a filter
+        cy.get('[id="service-name"]').type('Test');
+        cy.get('[type="submit"]').contains('Apply filter').click();
+
+        // Then the results are displayed
+        cy.get('[id="results"]').should('exist');
+        cy.get('[id="results"]').contains('test');
+        cy.get('tbody.govuk-table__body tr.govuk-table__row').its('length').then((length) => {
+            expect(length).to.equal(10);
+        })
+
+    })
+
+    it('can navigate back to homepage', () => {
+        const expectedPageUrl = "https://test.manage-family-support-services-and-accounts.education.gov.uk/Welcome";
+
+        // When I click the back button
+        cy.clickBackLink();
+
+        // Then I am navigated back to the homepage
+        cy.checkPageUrl(expectedPageUrl);
+
+    })
+
+    it('No results found page on filter', () => {
```

Comment:
nitpick: a massive nitpick, but can we standardise the casing used for `it()` names (e.g, no capital `N` in `No results...`, or change the others to be sentence case)

Reply:
Amended to be lower case inline with other tests